migrations/sql/R__stored_procedure_for_bond_etl.sql

```diff
@@ -0,0 +1,358 @@
+CREATE OR REPLACE FUNCTION mms_etl_bond_data() RETURNS void AS $$
+BEGIN
+declare
+
+		tmp_num_records integer;
+        tmp1 integer;
+        tmp2 integer;
+        tmp3 integer;
+  begin
+	--drop table if exists ETL_BOND;
+	CREATE TABLE IF NOT EXISTS ETL_BOND(
+		core_permit_id integer,
+		core_bond_id integer,
+		core_first_name varchar,
+		core_party_name varchar,
+		core_party_type varchar,
+		--local coalese secsec.cmp_nm is typed, secsec.first_nm,last_nm are autopoulated from permit
+		--secsec.cmp_nm is usually a 'companyName', or 'first last' or 'last, first' unsure how to parse.
+		core_payer_party_guid uuid,
+		etl_create_date TIMESTAMP,
+		etl_update_date TIMESTAMP,
+		-------Source columns we care about
+		sec_cid varchar NOT null unique , --save for reverse lookups
+		permit_no varchar NULL, -- used to lookup permit_id
+		mms_address varchar NULL, --preserving, but can't be converted.
+	--?? ignore bond.payer_party.address
+--		address varchar NULL,--?? Payer address
+--		city varchar NULL,--?? Street, Ciry, Prov are all mixed between addr1,2,3
+--		prov varchar NULL,
+--		post_cd varchar NULL,--?? Payer post
+		note1 varchar NULL,--?? Payer Contact Name
+		sec_amt numeric(12,2) NOT NULL, --bond.amount
+		core_bond_type_code varchar null,
+		sec_typ varchar NULL, -- mapped to bond.bond_type_code
+		descript varchar NULL,--bond.reference_number
+		"comment" varchar NULL, --bond.comment (joine secsec.comment1 and secsec.comment2)
+		invloc varchar NULL, --bond.institution_name
```

Comment:
What is `invloc`?

Reply:
mms's field name. I presume Investment Location.

Reply:
Ok, can we update its comment to reflect that? And maybe change the var name to `investment_location`?

Reply:
The comment describes the exact target column it will end up in, i don't see the value in parsing MMS's naming.